dewy/common/collection_embeddings.py

```diff
@@ -213,7 +214,8 @@ async def ingest(self, document_id: int, url: str) -> None:
                     INSERT INTO chunk (document_id, kind, text)
                     VALUES ($1, $2, $3);
                     """,
-                    [(document_id, "text", text_chunk) for text_chunk in text_chunks],
+                    # [(document_id, "text", bytes(text_chunk, 'utf-8').decode('utf-8', 'ignore')) for text_chunk in text_chunks],
+                    [(document_id, "text", text_chunk.encode('utf-8').decode('utf-8', 'ignore').replace("\x00", "\uFFFD")) for text_chunk in text_chunks],
```

Comment:
Handling invalid UTF8 and `\x00` which is valid, but not allowed by pg

Reply:
Fun. Feels like we may want to make that into a clearer utility at some point if this continues to be common. Please add a comment on that and maybe put in a helper function already... I suspect we'll want to remember why we're doing that at some point...